Templates/PrebuiltGem/template.json

```diff
@@ -3,7 +3,7 @@
     "origin": "The primary repo for PrebuiltGem goes here: i.e. http://www.mydomain.com",
     "license": "Apache-2.0 or MIT",
     "license_url": "https://github.com/o3de/o3de/blob/development/LICENSE.txt",
-    "display_name": "Prebuilt Gem Tenokate",
+    "display_name": "Prebuilt Gem Template",
```

Comment:
What was I on when checked this in?

Reply:
Ah it was a right hand keyboard alignment issue.
The 'n ' key is to the left of 'm'
The 'o' key is to the left of 'p'
The 'k' key is to the left of 'l'.

All the characters that were typed with the left hand are correct: 't', 'e', and 'a'